File: GetStartedScreen.dart
import 'package:flutter/material.dart';

class GetStartedScreen extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      body: Container(
        color: HexColor("#ffd43f"),
        child: Column(
          crossAxisAlignment: CrossAxisAlignment.stretch,
          mainAxisAlignment: MainAxisAlignment.spaceBetween,
          children: <Widget>[
            Expanded(
              child: Container(
                child: Image.asset(
                  "assets/images/banner.png",
                  fit: BoxFit.fitHeight,
                ),
              ),
            ),
            Container(
              padding: EdgeInsets.all(24.0),
              decoration: BoxDecoration(
                shape: BoxShape.rectangle,
                color: Colors.white,
                borderRadius: BorderRadius.only(
                  topLeft: Radius.circular(16),
                  topRight: Radius.circular(16),
                ),
              ),
              child: Column(
                crossAxisAlignment: CrossAxisAlignment.stretch,
                children: <Widget>[
                  Padding(
                    padding: EdgeInsets.only(top: 16),
                  ),
                  Text(
                    "Welcome to your\nteam calender,\nMargaret!",
                    style: TextStyle(fontSize: 32, fontWeight: FontWeight.bold),
                  ),
                  Padding(
                    padding: const EdgeInsets.symmetric(vertical: 32.0),
                    child: Text(
                      "Here you can see all your meetings and create new ones.",
                      style: TextStyle(color: Colors.black, fontSize: 16),
                    ),
                  ),
                  MaterialButton(
                    height: 48,
                    onPressed: () {},
                    color: HexColor("#6201ee"),
                    textColor: Colors.white,
                    shape: RoundedRectangleBorder(
                        borderRadius: BorderRadius.circular(5.0)),
                    child: Text("Get Started"),
                  ),
                ],
              ),
            ),
          ],
        ),
      ),
    );
  }
}

/*
* A Dart Utility class for Using Hex color string in Flutter
* Examples:
* color: HexColor("#ff4433")
* color: HexColor("ff4433")
* color: HexColor("#80ff4433")
* */
class HexColor extends Color {
  HexColor(String hexString) : super(_parseColor(hexString));

  static int _parseColor(String color) {
    try {
      color = color.toUpperCase().replaceAll("#", "");
      if (color.length == 6) {
        color = "FF" + color;
      }
    } on Exception catch (e) {
      return Colors.white.value;
    }
    return int.parse(color, radix: 16);
  }
}

File: GetStartedScreen.kt
package io.github.birjuvachhani.composeloginscreenui

import androidx.compose.Composable
import androidx.compose.unaryPlus
import androidx.ui.core.Text
import androidx.ui.core.dp
import androidx.ui.core.sp
import androidx.ui.foundation.DrawImage
import androidx.ui.foundation.shape.corner.RoundedCornerShape
import androidx.ui.graphics.Color
import androidx.ui.layout.*
import androidx.ui.material.Button
import androidx.ui.material.ContainedButtonStyle
import androidx.ui.material.MaterialTheme
import androidx.ui.material.surface.Surface
import androidx.ui.res.imageResource
import androidx.ui.text.TextStyle
import androidx.ui.text.font.FontFamily
import androidx.ui.text.font.FontWeight
import androidx.ui.tooling.preview.Preview

/*
 * Created by Birju Vachhani on 30 October 2019
 * Copyright © 2019 Compose Login Screen UI. All rights reserved.
 */

@Preview
@Composable
fun GetStartedScreen() {
    val backgroundColor = android.graphics.Color.parseColor("#ffd43f") // yellow background
    MaterialTheme {
        Surface(color = Color(backgroundColor)) {
            Column(
                modifier = Expanded,
                crossAxisAlignment = CrossAxisAlignment.Start,
                mainAxisAlignment = MainAxisAlignment.SpaceBetween
            ) {
                val image = +imageResource(R.drawable.banner2)
                // header image
                Container(modifier = Flexible(flex = 1f), expanded = true) {
                    DrawImage(image)
                }
                
                // bottom text part
                Surface(
                    color = Color(android.graphics.Color.WHITE),
                    shape = RoundedCornerShape(
                        topLeft = 16.dp,
                        bottomLeft = 0.dp,
                        bottomRight = 0.dp,
                        topRight = 16.dp
                    )
                ) {
                    Container(expanded = true, padding = EdgeInsets(all = 24.dp)) {
                        Column(
                            modifier = Expanded
                        ) {
                            Padding(padding = EdgeInsets(top = 8.dp)) {
                                Text(
                                    "Welcome to your\nteam calender,\nMargaret!",
                                    style = TextStyle(
                                        color = Color.Black,
                                        fontSize = 32.sp,
                                        fontWeight = FontWeight.Bold,
                                        fontFamily = FontFamily.SansSerif
                                    )
                                )
                            }
                            Padding(padding = EdgeInsets(top = 32.dp, bottom = 32.dp)) {
                                Text(
                                    "Here you can see all your meetings and create new ones.",
                                    style = TextStyle(
                                        color = Color.Black
                                    )
                                )
                            }
                            Button(
                                onClick = {},
                                children = {
                                    Container(expanded = true, height = 32.dp) {
                                        Text("Get Started", style = TextStyle(color = Color.White))
                                    }
                                },
                                style = ContainedButtonStyle(rippleColor = Color.White)
                            )
                        }
                    }
                }
            }

        }
    }
}
 